cpp/src/slice2cs/Gen.cpp

```diff
@@ -2662,24 +2704,29 @@ Slice::Gen::DispatcherVisitor::writeReturnValueStruct(const OperationPtr& operat
     if (operation->hasMarshaledResult())
     {
         _out << sp;
+        _out << nl << "/// <summary>Helper struct used to marshal the return value of " << opName << " operation."
+             << "</summary>";
         _out << nl << "public struct " << opName << "MarshaledReturnValue";
         _out << sb;
+        _out << nl << "/// <summary>The frame holding the marshaled response.</summary>";
         _out << nl << "public ZeroC.Ice.OutgoingResponseFrame Response { get; }";
 
+        _out << nl << "/// <summary>Construct a new <see cref=\"" << opName << "MarshaledReturnValue\"/> instance that";
+        _out << nl << "/// immediately marshals the response for the " << opName << "operation.</summary>";
```

Comment:
... marshals the return value of operation opName.

Reply:
fixed